sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithStateInitialStateSuite.scala

```diff
@@ -125,6 +129,29 @@ class InitialStateInMemoryTestClass
   }
 }
 
+class StatefulProcessorWithInitialStateProcTimerClass
```

Comment:
nit: could we add a small comment for each of these classes ? i.e. to explain their behavior for the initial state methods ?

Reply:
nit: maybe we should clarify - that we are trying to test for how initial state, timers and outputting rows might interact ? not sure if thats clear from the comment above ?